subprojects/scala/src/test/groovy/org/gradle/api/tasks/scala/ScalaDocSpec.groovy

```diff
@@ -16,21 +16,40 @@
 package org.gradle.api.tasks.scala
 
 import org.gradle.api.internal.AbstractTask
-import org.gradle.api.tasks.AbstractSpockTaskTest
+import org.gradle.api.internal.file.TestFiles
+import org.gradle.api.internal.project.taskfactory.TaskInstantiator
+import org.gradle.api.tasks.AbstractConventionTaskTest
+
 import org.gradle.util.WrapUtil
 
 import static org.gradle.api.tasks.compile.AbstractCompileTest.*
 
-class ScalaDocSpec extends AbstractSpockTaskTest {
+class ScalaDocSpec extends AbstractConventionTaskTest {
```

Comment:
I saw that `GroovydocTest` uses `AbstractConventionTaskTest` and took the opportunity to consolidate that